### PR TITLE
make release-artifacts

### PR DESCRIPTION
**Pull request by zirain**
**What type of PR is this?**

**What this PR does / why we need it**:

add `make release-artifacts` for release flow

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: hack/release-artifacts.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+VERSION=${VERSION:-"latest"}
+REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")
+OUT_BASE_PATH=${OUT_BASE_PATH:-"${REPO_ROOT}/out"}
+RELEASE_PATH="${OUT_BASE_PATH}/release-artifacts"
+CHART_PATH="${OUT_BASE_PATH}/charts"
+
+rm -rf "${RELEASE_PATH}"
+mkdir -p "${RELEASE_PATH}"
+
+BINS=("kurator")
```
Comment (hzxuzhonghu) [inline hack/release-artifacts.sh:19]: Not sure i understand what does BINS mean with an array

Seems this shoud always be kurator

File: hack/release-artifacts.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    echo "TAR BINARY: ${BIN}"
+    BIN_RELEASE="${BIN}_${VERSION}_${GOOS}-${GOARCH}.tar.gz"
+    pushd "${OUT_BASE_PATH}/${GOOS}-${GOARCH}/"
+        tar -zcvf "${BIN_RELEASE}" "./${BIN}"
+    popd
+    mv "${OUT_BASE_PATH}/${GOOS}-${GOARCH}/${BIN_RELEASE}" "${RELEASE_PATH}"
+done
+
+# copy charts
+cp -r "${CHART_PATH}"/*.tgz "${RELEASE_PATH}"
```
Comment (hzxuzhonghu) [inline hack/release-artifacts.sh:32]: What does it do

Comment (zirain) [reply hack/release-artifacts.sh:32]: copy all charts to release path, we can setup an action to auto upload it.

File: hack/release-artifacts.sh (hunk 1 above)
Comment (zirain) [reply hack/release-artifacts.sh:19]: yes, just in case there's another cli binary in the future?

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**